components/camera/server.go

```diff
@@ -53,8 +57,28 @@ func (s *subtypeServer) GetImage(
 		return nil, err
 	}
 
+	// Determine the mimeType we should try to use based on camera properties
 	if req.MimeType == "" {
-		req.MimeType = utils.MimeTypeRawRGBALazy
+		if s.imgType == nil {
```

Comment:
Since this camera server will be used for all cameras, I think you'll have to use a map of names in the struct, i.e. `imgTypes map[string]ImageType`.
And then in the body you'll do a 
```
if imgType, ok := s.imgTypes[req.Name]
if !ok {
  // calling the properties() func and putting the result in the map, then setting imgType
}
```

Reply:
Yup, you were right about this. Made a map